app/src/androidTest/java/com/github/h3lp3rs/h3lp/forum/ForumNewPostActivityTest.kt

```diff
@@ -39,10 +44,10 @@ class ForumNewPostActivityTest {
     fun addNewPostWorks() {
         // Create new post
         onView(withId(R.id.newPostCategoryDropdown))
-            .perform(ViewActions.replaceText(CATEGORY_TEST_STRING))
+            .perform(replaceText(CATEGORY_TEST_STRING))
```

Comment:
Just for better modularity, a CATEGORY_TEST val can be created with the corresponding category and passed to the forumOf function call during the setUp.

Reply:
Cf Alexandre's comment, the forum category should be put in a companion object, but this might break some stuff and need useless imports. This should be done in a refactoring round.